api/handlers.go

```diff
@@ -27,6 +27,9 @@ const (
 	// LimitBodySizeSmall defines a size limit for requests that we don't expect
 	// to contain a lot of data.
 	LimitBodySizeSmall = 4 * skynet.KiB
+	// LimitBodySizeLarge defines a size limit for requests that we expect to
+	// contain a lot of data.
+	LimitBodySizeLarge = 4 * skynet.MiB
```

Comment:
this will put a limit on the amount of skylinks covered by an API key right?
we should probably put an actual predefined limit on that and then measure it and update this value?

Reply:
sure, we can do that. for the moments I don't see a particular need for it - 4MB is a lot of data. if we see a use case in which a client wants to give public access to that many skylinks we can specify/change the limit. the main reason this is here is to prevent a malicious actor from running us OOM.